templates/bot/js/default/public/auth-start.html

```diff
@@ -18,6 +18,7 @@
         var currentURL = new URL(window.location);
         var clientId = currentURL.searchParams.get("clientId");
         var tenantId = currentURL.searchParams.get("tenantId");
+        var loginHint = currentURL.searchParams.get("loginHint");
```

Comment:
FYI: There're two `templates` folder:
1. TeamsFx/templates
2. TeamsFx/packages/fx-core/templates (fallback templates)

and under each `templates` folder, there're multiple public/auth-start.hml to be modified.